diffkemp/simpll/Config.cpp

```diff
@@ -107,13 +104,19 @@ Config::Config()
     }
 
     std::vector<std::string> debugTypes;
-    if (VerboseOpt) {
-        // Enable debugging output in passes
-        debugTypes.emplace_back(DEBUG_SIMPLL);
-    }
-    if (VerboseMacrosOpt) {
-        // Enable debugging output when finding macro differences
-        debugTypes.emplace_back(DEBUG_SIMPLL_MACROS);
+    if (VerbosityOpt > 0) {
+
+        // Enable debugging output in passes. Intended fallthrough
+        switch (VerbosityOpt) {
+        default:
+        case 3:
+            debugTypes.emplace_back(DEBUG_SIMPLL_VERBOSE_EXTRA);
+        case 2:
+            debugTypes.emplace_back(DEBUG_SIMPLL_VERBOSE);
```

Comment:
`[[fallthrough]]` attribute can be used to tell the compiler that the fallthough is intented: 
```suggestion
        case 3:
            debugTypes.emplace_back(DEBUG_SIMPLL_VERBOSE_EXTRA);
            [[fallthrough]];
        case 2:
            debugTypes.emplace_back(DEBUG_SIMPLL_VERBOSE);
            [[fallthrough]];
```